app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.java

```diff
@@ -75,55 +84,62 @@ public void unbind() {
         }
     };
 
-    private void setClearSearchVisible(@NonNull final Boolean clearVisible) {
-        Preconditions.checkNotNull(closeButton, "Close Button has not been bound");
+    private void setClearSearchVisible(final boolean clearVisible) {
+        checkNotNull(closeButton, "Close Button has not been bound");
 
         get(closeButton).setVisibility(clearVisible ? View.VISIBLE : View.GONE);
```

Comment:
I don't think we need both the `checkNotNull` and the `get` call on `closeButton`.